File: GGAndroid_3.0/core/support/rxjava/src/main/java/io/ganguo/rx/bus/RxEvent.java
package io.ganguo.rx.bus;


import androidx.core.util.Pair;

/**
 * rxbus event
 * Created by Lynn on 9/1/16.
 */

public class RxEvent extends Pair<String, Object> {
    public RxEvent(final String tag, final Object obj) {
        super(tag, obj);
    }
}

File: GGAndroid_4.0/core/support/support-rxbus/src/main/kotlin/io/ganguo/rxbus/RxEvent.kt
package io.ganguo.rxbus

import androidx.core.util.Pair

/**
 * <pre>
 *   @author : leo
 *   @time   : 2020/07/18
 *   @desc   : RxBus Event Object
 * </pre>
 */
class RxEvent(eventName: String, eventData: Any) : Pair<String, Any>(eventName, eventData)